### PR TITLE
Change the way of modifying version numbers (tiup)

### PR DESCRIPTION
**Pull request by glkappe**
或者是增加 tiup 方式，并提示生效方式，

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: faq/high-reliability-faq.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ category: FAQ
 
 ## 我们的安全漏洞扫描工具对 MySQL version 有要求，TiDB 是否支持修改 server 版本号呢？
 
-TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `conf/tidb.yml` 配置文件中的 `server-version` 来设置合适的版本号，以避免出现安全漏洞扫描不通过的问题。
+TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `tiup cluster edit-config clusterName` 配置文件中的 `server_configs - tidb - server-version` 来设置合适的版本号，并通过 reload -R tidb 生效，以避免出现安全漏洞扫描不通过的问题。
```
Comment (lonng) [inline faq/high-reliability-faq.md:13]: `server_configs - tidb - server-version` 用户很难看懂是什么意思，下面是不是更好
```yaml
server_configs:
  tidb:
    server-version: 'YOUR_VERSION_STRING'
```

Comment (glkappe) [reply faq/high-reliability-faq.md:13]: 是的，赞同

File: faq/high-reliability-faq.md [hunk 2]
```diff
@@ -10,7 +10,15 @@ category: FAQ
...
-TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `conf/tidb.yml` 配置文件中的 `server-version` 来设置合适的版本号，以避免出现安全漏洞扫描不通过的问题。
+TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `tiup cluster edit-config clusterName` 修改配置文件中以下部分来设置合适的版本号：
+
+```
+server_configs:
+  tidb:
+    server-version: 'YOUR_VERSION_STRING'
+```
+
+并通过 `reload -R tidb` 使得以上修改生效，以避免出现安全漏洞扫描不通过的问题。
```
Comment (TomShawn) [inline faq/high-reliability-faq.md:21]:
```suggestion
并通过 `reload -R tidb` 命令使得以上修改生效，以避免出现安全漏洞扫描不通过的问题。

```

**Comment by TomShawn** [comment]
@lonng PTAL again

File: faq/high-reliability-faq.md [hunk 3]
```diff
@@ -10,7 +10,16 @@ category: FAQ
...
+TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `tiup cluster edit-config clusterName` 修改配置文件中以下部分来设置合适的版本号：
+
+```
+server_configs:
+  tidb:
+    server-version: 'YOUR_VERSION_STRING'
+```
+
+并通过 `reload -R tidb` 命令使得以上修改生效，以避免出现安全漏洞扫描不通过的问题。
+
```
Comment (TomShawn) [inline faq/high-reliability-faq.md:22]:
```suggestion
```

File: faq/high-reliability-faq.md [hunk 4]
```diff
@@ -10,7 +10,15 @@ category: FAQ
 
 ## 我们的安全漏洞扫描工具对 MySQL version 有要求，TiDB 是否支持修改 server 版本号呢？
 
-TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `conf/tidb.yml` 配置文件中的 `server-version` 来设置合适的版本号，以避免出现安全漏洞扫描不通过的问题。
+TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `tiup cluster edit-config clusterName` 修改配置文件中以下部分来设置合适的版本号：
```
Comment (lonng) [inline faq/high-reliability-faq.md:13]:
```suggestion
TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiUP 部署集群时，可以通过 `tiup cluster edit-config clusterName` 修改配置文件中以下部分来设置合适的版本号：

```

File: faq/high-reliability-faq.md [hunk 5]
```diff
@@ -10,7 +10,15 @@ category: FAQ
...
-TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `conf/tidb.yml` 配置文件中的 `server-version` 来设置合适的版本号，以避免出现安全漏洞扫描不通过的问题。
+TiDB 在 v3.0.8 后支持修改 server 版本号，可以通过配置文件中的 [`server-version`](/tidb-configuration-file.md#server-version) 配置项进行修改。在使用 TiDB Ansible 部署集群时，同样可以通过 `tiup cluster edit-config clusterName` 修改配置文件中以下部分来设置合适的版本号：
+
+```
+server_configs:
+  tidb:
+    server-version: 'YOUR_VERSION_STRING'
+```
+
+并通过 `reload -R tidb` 命令使得以上修改生效，以避免出现安全漏洞扫描不通过的问题。
```
Comment (lonng) [inline faq/high-reliability-faq.md:21]:
```suggestion
并通过 `tiup cluster reload <cluster-name> -R tidb` 命令使得以上修改生效，以避免出现安全漏洞扫描不通过的问题。

```

**Pull request merged**